docs/content/manual/_incoming/2175-2332-harvester-pull-rancher-image-private-registry.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Harvester pull Rancher agent image from private registry
+---
+
+* Related issues: [#2175](https://github.com/harvester/harvester/issues/2175) [BUG] Harvester fails to pull Rancher agent image from private registry
+* Related issues: [#2332](https://github.com/harvester/harvester/issues/2332) [Backport v1.0] Harvester fails to pull Rancher agent image from private registry
+
+## Category: 
+* Virtual Machine
+
+## Verification Steps
+1. Create a harvester cluster and a ubuntu server. Make sure they can reach each other.
+1. On the ubuntu server, install docker and run the following commands.
+```
+mkdir -p certs
+openssl req \
+  -newkey rsa:4096 -nodes -sha256 -keyout certs/domain.key \
+  -addext "subjectAltName = DNS:myregistry.local" \
+  -x509 -days 365 -out certs/domain.crt
+sudo mkdir -p /etc/docker/certs.d/myregistry.local:5000
+sudo cp certs/domain.crt /etc/docker/certs.d/myregistry.local:5000/domain.crt
+docker run -d \
+  -p 5000:5000 \
+  --restart=always \
+  --name registry \
+  -v "$(pwd)"/certs:/certs \
+  -v "$(pwd)"/registry:/var/lib/registry \
+  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt \
+  -e REGISTRY_HTTP_TLS_KEY=/certs/domain.key \
+  registry:2
+docker pull rancher/rancher-agent:v2.6.5
+docker tag rancher/rancher-agent:v2.6.5 myregistry.local:5000/rancher/rancher-agent:v2.6.5
+docker push myregistry.local:5000/rancher/rancher-agent:v2.6.5
+```
```

Comment:
If you indent this then Hugo will consistently number the steps

Reply:
Appreciated for the reminder, double checked and remove the indent to fit the numbering format.
Aldo change the follow up numbering steps to the sequential manner.